src/menu/option-list.js

```diff
@@ -32,6 +32,18 @@ function OptionList(props: OptionListPropsT, ref: React.ElementRef<*>) {
     overrides.ListItem,
     StyledListItem,
   );
+  const getItem = item => {
+    return (
```

Comment:
Rather than writing the conditional in jsx, can this be refactored into two separate return statements please?